packages/react-sdk/src/css/categories.ts

```diff
@@ -10,7 +10,6 @@ const layout = [
   "justifyContent",
   "alignContent",
   // Grid
-  "placeContent",
```

Comment:
why removing it?

Reply:
Yes, because property was removed the generated list of properties and types no longer allow it in categories.